e2etest/scenario_helpers.go

```diff
@@ -836,10 +853,12 @@ func (s scenarioHelper) downloadFileContent(a asserter, options downloadContentO
 	defer retryReader.Close() // The client must close the response body when finished with it
 
 	destData, err := ioutil.ReadAll(retryReader)
+	a.AssertNoErr(err)
 	downloadResp.Body(azfile.RetryReaderOptions{})
 	return destData
 }
 
+//nolint
```

Comment:
there seems to be a ton of //nolint, what's the reason?

Reply:
I don't want to remove code which we are going to use in the near future. For example supporting S3, GCP Clients. The functions for them are there but `//nolint` will stop `golint` from marking them.

Reply:
And some times it gives false-positive as well.